source/getting-started.txt

```diff
@@ -0,0 +1,135 @@
+apps are available:
+
+- A mobile todo app written with Realm SDKs that syncs to an App Services App
+- A hosted todo web app using the GraphQL API or Realm Web SDK
+- An Event-driven :doc:`Database Trigger </triggers/database-triggers>` 
+  template that updates a view in a separate collection.
+
+Template apps are available for the following languages:
+
+- JavaScript (React Native)
+- Kotlin (Android)
+- Swift (iOS + SwiftUI)
+- ...and more!
+
+Checkout the template apps or visit the Tutorials section for a walkthrough
+of a template app.
+
+.. button:: Check out the Template Apps
+   :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_realm_template_apps
+
+.. _app-services-get-started-tutorials:
+
+Tutorials
+---------
+
+Unlike Template Apps, which provide a pre-built app to explore by 
+yourself, Tutorials provide a detailed step-by-step guide to developing 
+apps that use App Services features.
+
+.. card-group::
+   :style: extra-compact
+   :columns: 2
+
+   .. card::
+      :headline:  iOS with SwiftUI Template App Tutorial
+      :url: https://www.mongodb.com/docs/realm/tutorial/swiftui/ 
+      :icon: /images/icons/swift_sdk.svg
+      :icon-alt: Swift SDK
+      
+      Learn how to develop an iOS app that seamlessly syncs data using the
+      Realm Swift SDK and Device Sync. 
+
+   .. card::
+      :headline:  Android with Kotlin Template App Tutorial
+      :url: https://www.mongodb.com/docs/realm/tutorial/kotlin/ 
+      :icon: /images/icons/kotlin_sdk.svg
+      :icon-alt: Kotlin SDK
+      
+      Learn how to develop a Android app that seamlessly syncs data using 
+      the Realm Kotlin SDK and Device Sync. 
+
+   .. card::
+      :headline:  Xamarin with C# Template App Tutorial
+      :url: https://www.mongodb.com/docs/realm/tutorial/dotnet/ 
+      :icon: /images/icons/dot_net_sdk.svg
+      :icon-alt: Dot Net SDK
+      
+      Learn how to develop an iOS and Android compatible app that 
+      seamlessly syncs data using the Realm Xamarin SDK and Device Sync. 
+
+   .. card::
+      :headline:  React Native with JavaScript Template App Tutorial
+      :url: https://www.mongodb.com/docs/realm/tutorial/react-native/ 
+      :icon: /images/icons/react_native_sdk.svg
+      :icon-alt: React Native SDK
+      
+      Learn how to develop an iOS and Android compatible app that 
+      seamlessly syncs data using the Realm React Native SDK and Device
+      Sync. 
+
+   .. card::
+      :headline:  Flutter with Dart Template App Tutorial
+      :url: https://www.mongodb.com/docs/realm/tutorial/flutter/ 
+      :icon: /images/icons/flutter.svg
+      :icon-alt: Flutter SDK
+      
+      Learn how to develop an iOS and Android compatible app that 
+      seamlessly syncs data using the Realm Flutter SDK and Device Sync. 
+
+   .. card::
+      :headline: Using JWT for Client Authentication
+      :url: https://www.mongodb.com/docs/atlas/app-services/tutorial/jwt/
+      :icon: /images/icons/realm/user-authorization.svg
```

Comment:
replace this with an icon using the new branding. 

this one could work - https://mongodb.getbynder.com/web/1f0f4de208cca69e/icons/?mediaId=A73C1A45-580E-4FB7-BA156F2F5E05C8CB